frontend/packages/topology/integration-tests/support/step-definitions/topology/quick-search-topology.ts

```diff
@@ -15,3 +30,193 @@ Then('Display options dropdown, Filter by resource and Find by name fields are d
 Then('switch view is disabled', () => {
   cy.get(topologyPO.switcher).should('have.attr', 'aria-disabled', 'true');
 });
+
+Given('user created a workload and is at topology list view', () => {
+  navigateTo(devNavigationMenu.Add);
+  createGitWorkload(
+    'https://github.com/sclorg/nodejs-ex.git',
+    'nodejs-app',
+    'deployment',
+    'nodejs-ex-git-app',
+    false,
+  );
+  navigateTo(devNavigationMenu.Topology);
+  // eslint-disable-next-line promise/catch-or-return
+  cy.get('body').then(($body) => {
+    if ($body.find(topologyPO.quickSearchPO.listView).length !== 0) {
+      cy.get(topologyPO.quickSearchPO.toggleView).click();
+    }
+  });
+  cy.get(topologyPO.quickSearchPO.graphView).should('be.visible');
```

Comment:
there is a function in `topology-page` called `verifyUserIsInGraphView` that should do this, it should be better than repeating the same code.

also with a slight modification you should be able to get a function to verify list mode